packages/open-next/src/build/helper.ts

```diff
@@ -209,9 +209,12 @@ export function getOpenNextVersion(): string {
   return require(path.join(__dirname, "../../package.json")).version;
 }
 
-export function getNextVersion(nextPackageJsonPath: string): string {
-  const version = require(nextPackageJsonPath)?.dependencies?.next;
-  // require('next/package.json').version
+export function getNextVersion(appPath: string): string {
+  // We cannot just require("next/package.json") because it could be executed in a different directory
+  const nextPackageJsonPath = require.resolve("next/package.json", {
```

Comment:
There are two variables in this file named `nextPackageJsonPath` with different semantic meanings. This one appears to be the package.json path for the `next` package itself, while the other (L28) seems to represent the package.json path for the Nextjs app that OpenNext is running against.

There is also a function called `findNextPackageJsonPath` that seems to obtain the latter.

To someone unfamiliar with the codebase, this is a bit confusing. Might I suggest renaming these to clarify which is which? Maybe this instance could be `nextDependencyPackageJsonPath` or something along those lines? 

Happy to be corrected if there is anything I've missed.

Reply:
Yeah you're right, they should be different. I think it would make more sense to rename the other one `packageJsonPath` or maybe `appPackageJsonPath`